### PR TITLE
New issue templates forms

### PR DESCRIPTION
**Pull request by CaselIT**
Noting fancy. we could add checkbox and/or dropdowns if needed

**To test**: to go my fork. Link https://github.com/CaselIT/sqlalchemy/issues/new/choose

See docs at
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: To Reproduce
+      description: "
+Provide your [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/mcve) example here."
```
Comment (jvanasco) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:36]: I would add to the description:

If you need help creating one, you can model yours after the MCV code shared in one of our previous [well written bug reports](https://github.com/sqlalchemy/sqlalchemy/issues?q=is%3Aissue+label%3A%22great+mcve%22)

Comment (CaselIT) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:36]: Its referenced at the top, but we can add that here also.

Comment (jvanasco) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:36]: Yeah, I basically just copy/pasted your line to repeat it there. People are bad about code samples, so repeating it and just being "YOU CAN USE ANOTHER FORKING PERSON'S EXAMPLE CODE AS A TEMPLATE" is helpful.

Comment (CaselIT) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:36]: it was my reasoning too. will update

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+
+name: Create a bug report
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: [requires triage]
+body:
+  - type: markdown
+    attributes:
+      value: "
+**GUIDELINES FOR REPORTING BUGS**
```
Comment (zzzeek) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:11]: I'm trying this out at testgerrit:   https://github.com/sqlalchemy/testgerrit/issues/new?assignees=&labels=requires+triage&template=bug_report.yaml

this is GREAT!!! we can PUT WORDS THAT THEY HAVE TO READ!!! W LINKS!!! HOORAY!!!

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **zzzeek** to try to get revision 21beb5a44104ced9e43bb5342a4cdf5f5efcb02a of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 21beb5a44104ced9e43bb5342a4cdf5f5efcb02a: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2979

**Comment by sqla-tester** [comment]
**mike bayer** (zzzeek) wrote:

can't merge this fast enough!!!

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2979

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2979 has been **merged**. Congratulations! :)

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 2 above)
Comment (CaselIT) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:11]: I had also rigged by fork to test these.
>To test: to go my fork. Link https://github.com/CaselIT/sqlalchemy/issues/new/choose

it's quite neat

**Pull request closed without merging**